module/cyphersystem.js

```diff
@@ -171,10 +171,10 @@ Hooks.once("ready", async function() {
 
           totalModifier = skillRating + modifiedBy;
 
-          if (totalModifier == 1) totalModified = "eased";
-          if (totalModifier >= 2) totalModified = "eased by " + totalModifier + " steps";
-          if (totalModifier == -1) totalModified = "hindered";
-          if (totalModifier <= -2) totalModified = "hindered by " + Math.abs(totalModifier) + " steps";
+          if (totalModifier == 1) totalModified = game.i18n.localize("CYPHERSYSTEM.Eased").toLowerCase();
+		  if (totalModifier >= 2) totalModified = game.i18n.localize("CYPHERSYSTEM.EasedBy") + " " + totalModifier + " " + game.i18n.localize("CYPHERSYSTEM.Steps");
+		  if (totalModifier == -1) totalModified = game.i18n.localize("CYPHERSYSTEM.Hindered").toLowerCase();
```

Comment:
Are there instances in which `CYPHERSYSTEM.Eased` and `CYPHERSYSTEM.Hindered` are used with upper case? I’m not entirely confident that other languages capitalize those differently, so we should put the capitalization into the translation file, not hard code them here with `toLowerCase`.

Reply:
Yes, only one time, even if I do not remember when. I suspect it is on the chat card.
And `toLowerCase` is less complicated than using `string.charAt(0).toUpperCase() + string.slice(1)` to only capitalise the first letter.

Reply:
Unless, I just mixed `Eased/Hindered` with `Eased by/Hindered by`

Reply:
> Yes, only one time, even if I do not remember when. I suspect it is on the chat card.
> And `toLowerCase` is less complicated than using `string.charAt(0).toUpperCase() + string.slice(1)` to only capitalise the first letter.

Thing is, if it’s capitalized even once, we need a different string for that, or use `string.charAt(0).toUpperCase() + string.slice(1)` there once (if we are sure that this instance is capitalized in all languages). It’s better than to capitalize it in the string and then use `toLowerCase` every time.

Reply:
No worries we can do two localisation string in this case ^^

Reply:
Refactored translation